osu.Game/Rulesets/Edit/Checks/CheckPreviewTime.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Edit.Checks.Components;
+
+namespace osu.Game.Rulesets.Edit.Checks
+{
+    public class CheckPreviewTime : ICheck
+    {
+        public CheckMetadata Metadata => new CheckMetadata(CheckCategory.Timing, "Check Preview Time Consistency");
+
+        public IEnumerable<IssueTemplate> PossibleTemplates => new IssueTemplate[]
+        {
+            new IssueTemplatePreviewTimeConflict(this),
+            new IssueTemplateHasNoPreviewTime(this),
+        };
+
+        public IEnumerable<Issue> Run(BeatmapVerifierContext context)
+        {
+            var diffList = context.Beatmap.BeatmapInfo.BeatmapSet?.Beatmaps ?? new List<BeatmapInfo>();
+            int previewTime = context.Beatmap.BeatmapInfo.Metadata.PreviewTime;
+
+            if (previewTime == -1)
+            {
+                yield return new IssueTemplateHasNoPreviewTime(this).Create();
+            }
+
+            foreach (var diff in diffList)
+            {
+                if (diff.Equals(context.Beatmap.BeatmapInfo))
+                {
+                    continue;
+                }
+
+                if (diff.Metadata.PreviewTime != previewTime)
+                {
+                    yield return new IssueTemplatePreviewTimeConflict(this).Create(diff.DifficultyName, previewTime, diff.Metadata.PreviewTime);
+                }
+            }
+        }
+
+        public class IssueTemplatePreviewTimeConflict : IssueTemplate
+        {
+            public IssueTemplatePreviewTimeConflict(ICheck check)
+                : base(check, IssueType.Warning, "Audio preview time {1} doesn't match \"{0}\"'s preview time {2}")
```

Comment:
```suggestion
                : base(check, IssueType.Problem, "Audio preview time {1} doesn't match \"{0}\"'s preview time {2}")
```

[to match BeatmapVerifier](https://github.com/Naxesss/MapsetChecks/blob/8efbf76d17aa495732b2de0b75e33132169cecd4/Checks/AllModes/Timing/CheckPreview.cs#L50)